content/chainguard/chainguard-enforce/changelog.md

```diff
@@ -1,12 +1,15 @@
 ---
 title : "Chainguard Enforce Changelog"
+linktitle: "Changelog"
+aliases:
+- /chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog/
```

Comment:
This file comes from the Github workflow, so changes here will be lost. I think for this PR the easiest fix would be to leave the file in its original location, and we update the template that generates it, and the workflow separately?